vs.language.puppet/puppetDef.js

```diff
@@ -213,9 +213,7 @@ define(["require", "exports"], function (require, exports) {
                 [/^(\s*)([a-z_]\w*[!?=]?)/, ['white', { cases: { 'for|until|while': { token: 'keyword.$2', bracket: '@open', next: '@dodecl.$2' }, '@declarations': { token: 'keyword.$2', bracket: '@open', next: '@root.$2' }, 'end': { token: 'keyword.$S2', bracket: '@close', next: '@pop' }, '@keywords': 'keyword', '@builtins': 'predefined', '@default': 'identifier' } }]],
                 [/[a-z_]\w*[!?=]?/, { cases: { 'if|unless|while|until': { token: 'keyword.$0x', bracket: '@open', next: '@modifier.$0x' }, 'for': { token: 'keyword.$2', bracket: '@open', next: '@dodecl.$2' }, '@linedecls': { token: 'keyword.$0', bracket: '@open', next: '@root.$0' }, 'end': { token: 'keyword.$S2', bracket: '@close', next: '@pop' }, '@keywords': 'keyword', '@builtins': 'predefined', '@default': 'identifier' } }],
                 [/[A-Z][\w]*[!?=]?/, 'constructor.identifier'],
-                [/\$[\w]*/, 'global.constant'],
-                [/@[\w]*/, 'namespace.instance.identifier'],
-                [/@@[\w]*/, 'namespace.class.identifier'],
+                [/\$\s*[a-zA-Z_\$][\w\$]*/, 'annotation'],

```

Comment:
Changes color to annotation stole from java which will be a different color and stickout in IDE